### PR TITLE
Fixed the display issue of multimeter layout on tablet devices …

### LINKED ISSUE #2118: Multimeter: Layout looks weird on tablet devices
**Actual Behaviour**

The multimeter looks weird on tablet devices, the measured value is partly hidden.

**Expected Behaviour**

The layout should look more pleasing (larger knob or a margin around the whole multimeter) and the value should be completely visible.

**Steps to reproduce it**

1. Start app on tablet device
1. Select multimeter

**Screenshots of the issue**

Screenshot_20200610-224910

**Would you like to work on the issue?**

Maybe..

### PR DESCRIPTION
**Pull request by Girish-Bharadwaj**
Fixed display issue of multimeter  layout on tablet devices

Fixes #2118 

**Changes**: Increased margin of the multimeter and increased height of the display box of multimeter layout by adding new dimens resource file for xhdpi.Also increased the size of the knob along with text in it **without hardcoding** them.

**Screenshot for the changes**: 
Screenshot (131)

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [ ] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**:
[app-fdroid-debug.zip](https://github.com/fossasia/pslab-android/files/5318865/app-fdroid-debug.zip)

### CODE REVIEW

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Hello @Girish-Bharadwaj , I assume this is your first PR for this repo.

First of all, use a meaningful name for the PR title. Secondly, read the PR template and try to fill it. It's asking for screenshots to view the results and an APK to test the app with the changes.

Please update with necessary changes.

**Comment by Girish-Bharadwaj** [comment]
> Hello @Girish-Bharadwaj , I assume this is your first PR for this repo.
> 
> First of all, use a meaningful name for the PR title. Secondly, read the PR template and try to fill it. It's asking for screenshots to view the results and an APK to test the app with the changes.
> 
> Please update with necessary changes.

Sir I have updates the changes as you said.Sorry for inconvinience as it was my first PR and thank you for guiding me Sir.

**Comment by CloudyPadmal** [comment]
Hello @Girish-Bharadwaj , Is it possible to make the center knob a bit bigger. As you can see from the screenshot, lot of space is left unused,

**Comment by Girish-Bharadwaj** [comment]
> Hello @Girish-Bharadwaj , Is it possible to make the center knob a bit bigger. As you can see from the screenshot, lot of space is left unused,

Hello Sir,I have made changes as you said.

**Comment by CloudyPadmal** [comment]
Okay, now the knob looks good. But the labels around it also needs to be mapped as it was in the smaller layout.

**Comment by Girish-Bharadwaj** [comment]
> Okay, now the knob looks good. But the labels around it also needs to be mapped as it was in the smaller layout.

Sir made the changes as you said

File: app/src/main/res/layout-xhdpi/activity_multimeter.xml [hunk 1]
```diff
@@ -283,7 +283,10 @@
                 android:textStyle="bold"
                 app:layout_constraintCircle="@id/knobs"
                 app:layout_constraintCircleAngle="@string/multimeter_angle_12"
-                app:layout_constraintCircleRadius="@dimen/multimeter_knobcircle_radius_2_xl" />
+                app:layout_constraintCircleRadius="@dimen/multimeter_knobcircle_radius_2_xhdpi"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                tools:layout_editor_absoluteY="290dp"/>
```
Comment (CloudyPadmal) [inline app/src/main/res/layout-xhdpi/activity_multimeter.xml:289]: You might have missed to add this one to dimens.xml

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Looks almost good :+1:

File: app/src/main/res/layout-xhdpi/activity_multimeter.xml (hunk 1 above)
Comment (Girish-Bharadwaj) [reply app/src/main/res/layout-xhdpi/activity_multimeter.xml:289]: > Looks almost good

Sorry sir,actually it was added by mistake.It was not necessary.Removed it Sir.

**Review by CloudyPadmal — APPROVED** [review]
Looks good. For this one, I'll squash the commits while merging. From next time onward, squash the commits into one. (right now there are 5)

**Comment by Girish-Bharadwaj** [comment]
> Looks good. For this one, I'll squash the commits while merging. From next time onward, squash the commits into one. (right now there are 5)

OK Sir.Thank you so much for guiding me through my first PR.

**Pull request merged**